### PR TITLE
Add WebDriver conformance updates for 108 release notes

### PR DESCRIPTION
**Pull request by nchevobbe**

### CODE REVIEW

**Comment by nchevobbe** [comment]
cc @whimboo @juliandescottes @lutien

File: files/en-us/mozilla/firefox/releases/108/index.md [hunk 1]
```diff
@@ -69,8 +69,17 @@ This article provides information about the changes in Firefox 108 that will aff
 
 #### WebDriver BiDi
 
+- Following a [change in the specification](https://github.com/w3c/webdriver-bidi/pull/259), log entry level `"warning"` was renamed to `"warn"` ([bug 1797115](https://bugzilla.mozilla.org/show_bug.cgi?id=1797115))
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/108/index.md:72]: Just an FYI; there's a macro you may use in this repo for your convenience:

```markdown
<!-- markdown link -->
[bug 1797115](https://bugzilla.mozilla.org/show_bug.cgi?id=1797115)

<!-- shorthand / macro -->
{{bug(1797115)}}
```

File: files/en-us/mozilla/firefox/releases/108/index.md [hunk 2]
```diff
@@ -69,8 +69,17 @@ This article provides information about the changes in Firefox 108 that will aff
...
+- When using `script.evaluate` and `script.callFunction` without sandbox name, the evaluation will be done using the default realm ([bug 1793589](https://bugzilla.mozilla.org/show_bug.cgi?id=1793589))
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/108/index.md:73]:
```suggestion
- When using `script.evaluate` and `script.callFunction` without a sandbox name, the evaluation will be done using the default realm ([bug 1793589](https://bugzilla.mozilla.org/show_bug.cgi?id=1793589))
```

File: files/en-us/mozilla/firefox/releases/108/index.md [hunk 3]
```diff
@@ -69,8 +69,17 @@ This article provides information about the changes in Firefox 108 that will aff
...
+- Added support for the `browsingContext.domContentLoaded` event ([bug 1756610](https://bugzilla.mozilla.org/show_bug.cgi?id=1756610))
+
 #### Marionette
 
+- Added support for `tiltX`, `tiltY` and `twist` on touch event ([bug 1793832](https://bugzilla.mozilla.org/show_bug.cgi?id=1793832))
+- Fixed a bug where "Get Element Text" wasn't returning the element text for pretty-printed XML ([bug 1794099](https://bugzilla.mozilla.org/show_bug.cgi?id=1794099))
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/108/index.md:79]:
```suggestion
- Fixed a bug where retrieving element text content for pretty-printed XML was failing ([bug 1794099](https://bugzilla.mozilla.org/show_bug.cgi?id=1794099))
```

**Comment by bsmth** [comment]
Thanks @nchevobbe, I've left a couple of comments. Let me know if you'd like me to wait for the people you've tagged in CC to have a look before merging 👀

File: files/en-us/mozilla/firefox/releases/108/index.md (hunk 2 above)
Comment (juliandescottes) [reply files/en-us/mozilla/firefox/releases/108/index.md:73]: To be accurate, the change is really about using an empty string as sandbox name. If you didn't provide a sandbox name, we were already using the default realm. But with `sandbox=""` we used to create sandbox named `""` (empty string). 

```suggestion
- When using `script.evaluate` and `script.callFunction` with a sandbox name equal to an empty string, the evaluation will be done using the default realm ([bug 1793589](https://bugzilla.mozilla.org/show_bug.cgi?id=1793589))
```

File: files/en-us/mozilla/firefox/releases/108/index.md (hunk 3 above)
Comment (whimboo) [inline files/en-us/mozilla/firefox/releases/108/index.md:79]: Lets use the API endpoint of Marionette:

```suggestion
- Fixed a bug where `WebDriver:GetElementText` wasn't returning the element text for pretty-printed XML ([bug 1794099](https://bugzilla.mozilla.org/show_bug.cgi?id=1794099))
```

File: files/en-us/mozilla/firefox/releases/108/index.md [hunk 4]
```diff
@@ -69,8 +69,17 @@ This article provides information about the changes in Firefox 108 that will aff
...
+- `HTMLDocument` are no longer serialized as a WebElement reference ([bug 1793920](https://bugzilla.mozilla.org/show_bug.cgi?id=1793920))
```
Comment (whimboo) [inline files/en-us/mozilla/firefox/releases/108/index.md:80]:
```suggestion
- `HTMLDocument` is no longer serialized as a `WebElement` reference ([bug 1793920](https://bugzilla.mozilla.org/show_bug.cgi?id=1793920))
```

File: files/en-us/mozilla/firefox/releases/108/index.md [hunk 5]
```diff
@@ -69,8 +69,17 @@ This article provides information about the changes in Firefox 108 that will aff
 
 #### WebDriver BiDi
 
+- Following a [change in the specification](https://github.com/w3c/webdriver-bidi/pull/259), log entry level `"warning"` was renamed to `"warn"` ([bug 1797115](https://bugzilla.mozilla.org/show_bug.cgi?id=1797115))
+- When using `script.evaluate` and `script.callFunction` without sandbox name, the evaluation will be done using the default realm ([bug 1793589](https://bugzilla.mozilla.org/show_bug.cgi?id=1793589))
+- Added support for the `browsingContext.domContentLoaded` event ([bug 1756610](https://bugzilla.mozilla.org/show_bug.cgi?id=1756610))
+
 #### Marionette
 
+- Added support for `tiltX`, `tiltY` and `twist` on touch event ([bug 1793832](https://bugzilla.mozilla.org/show_bug.cgi?id=1793832))
```
Comment (whimboo) [inline files/en-us/mozilla/firefox/releases/108/index.md:78]:
```suggestion
- Added support for the `tiltX`, `tiltY` and `twist` properties of pointer actions for `WebDriver:PerformActions` ([bug 1793832](https://bugzilla.mozilla.org/show_bug.cgi?id=1793832))
```

**Review by whimboo — COMMENTED** [review]
Thanks @nchevobbe. I've a couple of comments as well. Otherwise agree to what @bsmth already mentioned especially the macro.

**Comment by whimboo** [comment]
Here the list of changes that are release note worthy for the Firefox 108 release:

https://bugzilla.mozilla.org/buglist.cgi?status_whiteboard=[webdriver%3Arelnote]&resolution=FIXED&query_format=advanced&v1=fixed&j_top=OR&status_whiteboard_type=substring&o1=equals&f1=cf_status_firefox108&list_id=16318283

File: files/en-us/mozilla/firefox/releases/108/index.md (hunk 1 above)
Comment (nchevobbe) [reply files/en-us/mozilla/firefox/releases/108/index.md]: ah nice, I switched to the macro

**Comment by nchevobbe** [comment]
thanks all for the suggestions, I think I addressed all the comments

File: files/en-us/mozilla/firefox/releases/108/index.md [hunk 6]
```diff
@@ -69,8 +69,17 @@ This article provides information about the changes in Firefox 108 that will aff
 
 #### WebDriver BiDi
 
+- Following a [change in the specification](https://github.com/w3c/webdriver-bidi/pull/259), log entry level `"warning"` was renamed to `"warn"` ({{bug(1797115)}})
+- When using `script.evaluate` and `script.callFunction` with a sandbox name equal to an empty string, the evaluation will be done using the default realm ({{bug(1793589)}})
```
Comment (whimboo) [inline files/en-us/mozilla/firefox/releases/108/index.md:73]: I wonder if we should add `now`:

```suggestion
- When using `script.evaluate` and `script.callFunction` with a sandbox name equal to an empty string, the evaluation will now be done using the default realm ({{bug(1793589)}})
```

**Review by whimboo — COMMENTED** [review]
Looks fine to me. Just a very minor nit - not sure if that's worth :)

File: files/en-us/mozilla/firefox/releases/108/index.md (hunk 6 above)
Comment (nchevobbe) [reply files/en-us/mozilla/firefox/releases/108/index.md:73]: sure :)

**Comment by whimboo** [comment]
Thanks @nchevobbe! It looks all fine to me as well.

@bsmth feel free to merge. Thanks.

**Comment by bsmth** [comment]
Thanks, everyone. Merging shortly 👍🏻

**Review by bsmth — APPROVED** [review]

**Pull request merged**